src/pybind/mgr/dashboard/rgw.py

```diff
@@ -0,0 +1,45 @@
+# -*- code: utf-8 -*-
+
+import json
+import re
+import rados
+from remote_view_cache import RemoteViewCache
+
+import logging
+
+log = logging.getLogger("dashboard")
+
+class RGWDaemons(RemoteViewCache):
+
+    def _get(self):
+        daemons = self.get_daemons()
+        return {
+            'daemons': daemons,
+        }
+
+    def get_daemons(self):
+        daemons = []
+        for server in self._module.list_servers():
+            for service in server['services']:
+                if service['type'] == 'rgw':
+                    metadata = self._module.get_metadata('rgw', service['id'])
+                    status = self._module.get_daemon_status('rgw', service['id'])
+                    try:
+                        status = json.loads(status['json'])
+                    except:
+                        status = {}
```

Comment:
Let's do a `log.warn` here to say the `service['id']` which had invalid status json